### PR TITLE
Plugins cta

### PR DESCRIPTION
**Pull request by kamiiiel**
## :id: Reference related issue. 

https://gravitee.atlassian.net/browse/APIM-1685

### CODE REVIEW

File: gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/GraviteeLicenseServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author Antoine CORDIER (antoine.cordier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Component
+public class GraviteeLicenseServiceImpl implements GraviteeLicenseService {
```
Comment (a-cordier) [inline gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/GraviteeLicenseServiceImpl.java:38]: This service is already provided by gravitee-node :)

Comment (a-cordier) [reply gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/GraviteeLicenseServiceImpl.java:38]: It's NodeLicenseService

Comment (a-cordier) [reply gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/GraviteeLicenseServiceImpl.java:38]: see https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/installation/GraviteeLicenseResource.java

**Review by leleueri — CHANGES_REQUESTED** [review]
In addition of the comment made by Antoine, can you plkease rebase on 4.0.x branch to merge on 4..0.x first and apply the "apply-to-master" label on this PR ?

**Pull request merged**